installer/bundle/bundle_skel.sh

```diff
@@ -759,11 +732,21 @@ then
     echo "Failed: could not extract the install bundle."
     cleanup_and_exit ${STATUS}
 fi
+[ -n "${forceFlag}" ] && FORCE="--force" || FORCE=""
+[ -n "${debugMode}" ] && DEBUG="--debug" || DEBUG=""
+./bundles/${SCX_INSTALLER} --extract $FORCE $DEBUG
```

Comment:
I'm not sure why you're extracting the shell bundle like this. Why not leave the shell bundle in the list of extracted files as is (as a `.sh` file), and then just run it when you need to for installation or upgrade?

If they really did want SCX extracted, then they can run that shell bundle with --extract afterwards. But given that the bundle skeleton file should just execute the SCX shell bundle as a `.sh` file, why not just leave it in that form?